src/btree/bt_page.c

```diff
@@ -121,6 +121,43 @@ __wt_page_alloc(
     return (0);
 }
 
+/*
+ * __page_inmem_tombstone --
+ *     Create the actual update for a tombstone.
+ */
+static int
+__page_inmem_tombstone(
+  WT_SESSION_IMPL *session, WT_CELL_UNPACK_KV *unpack, WT_UPDATE **updp, size_t *sizep)
+{
+    WT_DECL_RET;
+    WT_UPDATE *tombstone;
+    size_t size, total_size;
+
+    size = 0;
+    *sizep = 0;
+
+    tombstone = NULL;
+    total_size = 0;
+
+    WT_ASSERT(session, WT_TIME_WINDOW_HAS_STOP(&unpack->tw));
+
+    WT_ERR(__wt_upd_alloc_tombstone(session, &tombstone, &size));
```

Comment:
I think `WT_RET` is enough here (no need for `err:` label).

Reply:
[bfbb3ff](https://github.com/wiredtiger/wiredtiger/pull/10959/commits/bfbb3ffd8e1972aa5c47de88a3faaf95ddecb7e4)